internal/manifests/manifestutils/labels.go

```diff
@@ -90,6 +89,6 @@ func SelectorMatchLabels(instance metav1.ObjectMeta, component string) map[strin
 		"app.kubernetes.io/instance":   naming.Truncate("%s.%s", 63, instance.Namespace, instance.Name),
 		"app.kubernetes.io/part-of":    "opentelemetry",
 		"app.kubernetes.io/component":  component,
-		"app.kubernetes.io/name":       fmt.Sprintf("%s-monitoring", instance.Name),
+		"app.kubernetes.io/name":       naming.ServiceMonitorSelectorName(instance.Name),
```

Comment:
I think we could just reuse [this method](https://github.com/open-telemetry/opentelemetry-operator/blob/00d2a0472fa9ce03f3b777899069a1f56f244846/internal/naming/main.go#L113-L116)

Reply:
bump on this, no need to define a new method IMO